### PR TITLE
pkg: add windows cpu utils support

### PR DESCRIPTION
**Pull request by AndrewDi**
### What problem does this PR solve?
Support tidb-tools to build on windows.

### What is changed and how it works?
Implement `GetCPUPercentage()` on Windows.

### Check List 

Tests

 - No code

### CODE REVIEW

**Comment by shenli** [comment]
@AndrewDi Thanks!

**Comment by IANTHEREAL** [comment]
@leoppro  PTAL

File: pkg/utils/cpu_windows.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	"time"
+)
+
+var (
+	lastInspectUnixNano int64
+	lastCPUUsageTime    int64
+)
+
+// GetCPUPercentage calculates CPU usage and returns percentage in float64(e.g. 2.5 means 2.5%).
+func GetCPUPercentage() float64 {
```
Comment (zier-one) [inline pkg/utils/cpu_windows.go:28]: how can we test it? all of our test servers(ci) are running at Linux.
do you have any idea?

Comment (AndrewDi) [reply pkg/utils/cpu_windows.go:28]: @leoppro I can just test on my windows, but it seems like `GetCPUPercentage` didn't have any usage now.

**Comment by zier-one** [comment]
LGTM

**Comment by zier-one** [comment]
LGTM

**Comment by IANTHEREAL** [comment]
LGTM

**Pull request merged**